CRM/Core/BAO/CustomGroup.php

```diff
@@ -1264,6 +1264,11 @@ public static function setDefaults($groupTree, &$defaults, $viewMode = FALSE, $i
                 }
               }
               else {
+                if ($field['html_type'] === 'Autocomplete-Select') {
+                  $checkedValue = array_filter(explode(CRM_Core_DAO::VALUE_SEPARATOR, $value));
```

Comment:
It's usually preferable to use `CRM_Utils_Array::explodePadded` because if the value is already an array it won't cause errors. Just have to cast it in case the output is null.
```suggestion
                  $checkedValue = array_filter((array) \CRM_Utils_Array::explodePadded($value));
```